kie-spring-boot/jbpm-spring-boot/pom.xml

```diff
@@ -0,0 +1,193 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  ~ Copyright 2017 Red Hat, Inc. and/or its affiliates.
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  -->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
We definitely need a test for this module. As it is now I think we cannot assume it just works as expected. I will help with creating tests but if you have one that you used when doing this pr please include it.

Reply:
@tsurdilo the only test I have is that I was able to build standalone application with this module and I was able to run it with spring boot. I would be glad to prepare any additional test but I have no in my mind :(

Reply:
@rstancel I think we could add all sorts of unit and integration tests with @WebMvcTest and @SpringBootTest. Having this module without any tests imo does not prove to community that it works as well as does not show any capabilities of why they should consider using it. WDYT? 
If you don't mind we could work together on your branch for this PR to add some tests? If you can please make sure I have write to your pr branch (i should have it already) and maybe ping me on PR whenever you can so we could get this up and running?